packages/text-sets/src/raw/step.json

```diff
@@ -1908,13 +1908,13 @@
               "lGap": 0
             }
           },
-          "fontSize": 24,
+          "fontSize": 23,
           "backgroundColor": { "color": { "r": 196, "g": 196, "b": 196 } },
           "lineHeight": 1.3,
           "textAlign": "center",
           "padding": { "locked": true, "horizontal": 0, "vertical": 0 },
           "type": "text",
-          "content": "<span style=\"color: #444; letter-spacing: 0.04em\">Quick Tutorial for Beginners</span>",
+          "content": "<span style=\"color: #444; letter-spacing: 0.02em\">Quick Tutorial for Beginners</span>",
           "fontWeight": 400,
```

Comment:
```suggestion
          "fontWeight": 300,
```

Reply:
Is this what you recommend - https://github.com/GoogleForCreators/web-stories-wp/pull/10886/files#r825824820 ?

Reply:
`font-weight` is actually part of `content`, e.g.

```
"content": "<span style=\"font-weight: 300; color: #444; letter-spacing: 0.02em\">Quick Tutorial for Beginners</span>",
```

I'm not sure where `fontWeight` is coming from 🤔  We might need to remove that. I'll look into it.

Reply:
See #10922 for removing those.